lib/flagship/features.rb

```diff
@@ -0,0 +1,17 @@
+class Flagship::Features < Array
+  def tagged(tags)
+    Flagship::Features.new(select{|feature| tags.any?{|tag, val| feature.tags[tag] == val}})
```

Comment:
I think using `any?` here makes confusion.

When there's a code like below, I think the `features` should have both of `tag_a: true` and `tag_b: true` by analogy of `ActiveRecord`.

```rb
Flagship.features.tagged(tag_a: true, tag_b: true)
```

So I suggest:

* Remove current `tagged` method
* Rename current `tagged_all` method to `tagged`

What do you think?